controller/controller.js

```diff
@@ -59,3 +60,9 @@ exports.getCommentsByArticleId = (req, res, next) => {
 		})
 		.catch(next);
 };
+
+exports.getAllUsers = (req, res, next) => {
+  selectAllUsers().then((users) => {
+		res.status(200).send(users);
```

Comment:
We should be sending this back on an object with a descriptive key. You will need to update your tests accordingly, which means making sure they fail first :)